lib/BannerPlugin.js

```diff
@@ -1,40 +1,51 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
-*/
-var ConcatSource = require("webpack-sources").ConcatSource;
-var ModuleFilenameHelpers = require("./ModuleFilenameHelpers");
+ MIT License http://www.opensource.org/licenses/mit-license.php
+ Author Tobias Koppers @sokra
+ */
+
+"use strict";
+
+const ConcatSource = require("webpack-sources").ConcatSource;
+const ModuleFilenameHelpers = require("./ModuleFilenameHelpers");
 
 function wrapComment(str) {
-	if(str.indexOf("\n") < 0) return "/*! " + str + " */";
-	return "/*!\n * " + str.split("\n").join("\n * ") + "\n */";
+	if(!str.includes("\n")) return `/*! ${str} */`;
+	return `/*!\n * ${str.split("\n").join("\n * ")}\n */`;
 }
 
-function BannerPlugin(options) {
-	if(arguments.length > 1)
-		throw new Error("BannerPlugin only takes one argument (pass an options object)");
-	if(typeof options === "string")
-		options = {
-			banner: options
-		};
-	this.options = options || {};
-	this.banner = this.options.raw ? options.banner : wrapComment(options.banner);
-}
-module.exports = BannerPlugin;
+class BannerPlugin {
+	constructor(options) {
+		if(arguments.length > 1)
+			throw new Error("BannerPlugin only takes one argument (pass an options object)");
+		if(typeof options === "string")
+			options = {
+				banner: options
+			};
+		this.options = options || {};
+		this.banner = this.options.raw ? options.banner : wrapComment(options.banner);
+	}
 
-BannerPlugin.prototype.apply = function(compiler) {
-	var options = this.options;
-	var banner = this.banner;
+	apply(compiler) {
+		const options = this.options;
```

Comment:
Hey would you mind changing these two consts to `let` for now. We haven't solidified our eslint styleguide yet, so for right now pattern is 

all const for non-object primitives (that you intend to keep immutable), and let for all object based primitives (array, object etc.), and const is appropriate for `require()`